docs/core/porting/cppcli.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Tools for porting to .NET Core
+description: Learn about porting C++/CLI projects to .NET Core
+author: mjrousos
+ms.author: 
+ms.date: 12/12/2019
+---
+
+# How to port a C++/CLI project to .NET Core
+
+Beginning with .NET Core 3.1 and Visual Studio 2019 16.4, C++/CLI projects can be updated to target .NET Core. This allows Windows desktop applications which may use C++/CLI projects as an interop layer between managed and native code to be ported to .NET Core. This article describes how to port C++/CLI projects from .NET Framework to .NET Core 3.1.
+
+## C++/CLI .NET Core limitations
+
+There are some important limitations to porting C++/CLI projects to .NET Core compared to C# or VB projects:
+
+* C++/CLI support for .NET Core is Windows only.
+* C++/CLI projects cannot target .NET Standard, only .NET Core (or, of course, .NET Framework).
+* C++/CLI projects do not support the new SDK-style project file format. Instead, even when targeting .NET Core, C++/CLI projects use the existing vcxproj file format.
+  * Because of this, C++/CLI projects cannot multitarget multiple .NET platforms. If you need to build a C++/CLI project for both .NET Framework and .NET Core, use separate project files.
+* .NET Core does not support /clr:pure or /clr:safe compilation, only the new /clr:netcore option (which is equivalent to /clr for .NET Framework).
+
+## C++/CLI porting process
+
+To port a C++/CLI project to .NET Core, make the following changes to the vcxproj project file:
+
+1. Replace `<CLRSupport>true</CLRSupport>` properties with `<CLRSupport>NetCore</CLRSupport>`.
+    1. Be aware that this property is often in configuration- and platform-specific property groups, so it may need to be replaced in multiple places.
```

Comment:
```suggestion
   This property is often in configuration-specific and platform-specific property groups, so it may need to be replaced in multiple places.
```